pkgs/development/python-modules/qiskit-aer/default.nix

```diff
@@ -0,0 +1,53 @@
+{ stdenv
+, isPy3k
+, buildPythonPackage
+, fetchFromGitHub
+, cmake
+, blas
+, liblapack
+, scikit-build
+, cython
+, numpy
+, qiskit-terra
+}:
+
+buildPythonPackage rec {
+  pname = "qiskit-aer";
+  version = "0.1.1";
+
+  disabled = !isPy3k;
+
+  src = fetchFromGitHub {
```

Comment:
Why not `fetchPyPI`? Please add an inline comment.

Reply:
added